app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    _ = [Person(human["name"], human["age"]) for human in people]
```

Comment:
it is better to use for loops for creating smth, list comprehension is used for creating list

Reply:
Yep, my mistake)